ray-operator/controllers/ray/rayjob_controller.go

```diff
@@ -142,6 +142,11 @@ func (r *RayJobReconciler) Reconcile(ctx context.Context, request ctrl.Request)
 		if shouldUpdate := r.updateStatusToSuspendingIfNeeded(ctx, rayJobInstance); shouldUpdate {
 			break
 		}
+		if pastActiveDeadline(rayJobInstance) {
+			r.Log.Info("The RayJob has passed the activeDeadlineSeconds. Transition the status to `Complete`.", "StartTime", rayJobInstance.Status.StartTime, "ActiveDeadlineSeconds", *rayJobInstance.Spec.ActiveDeadlineSeconds)
+			rayJobInstance.Status.JobDeploymentStatus = rayv1.JobDeploymentStatusComplete
```

Comment:
Do you think exceeding deadline seconds warrants a new status `DeadlineExceeded`?

Reply:
If not -- if user sees status `Complete`, are the ray-operator logs the only way to know if a job successfully completed or timed out?

Reply:
I will improve the observability of `Complete` before KubeRay v1.1.0 release. Last week, I chatted with the [Flyte](https://github.com/flyteorg/flyte) community, and they require sufficient observability to distinguish between "Succeeded" and "Failed" statuses, including the reasons for each (e.g. backoffLimit, app-level bug, DeadlineExceeded).

Reply:
@Yicheng-Lu-llll will take this issue.